src/app/js/lib/components/FieldDisplayInListInput.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
over splitting ? All the component you added could be rewritten as one HOC passing the component to the component property of Field.
```js
Component => ({ label, name, fullWidth}) => (
    <Field component={Component} label={label} ...  />
);
```
And then FormCheckBoxField, FormatEdition, etc... could use the HOC directly in their export.